packages/squiggle-lang/__tests__/SquiggleLibrary/SquiggleLibrary_FunctionRegistryLibrary_test.res

```diff
@@ -0,0 +1,78 @@
+open Jest
+open Expect
+open Reducer_TestHelpers
+
+let expectEvalToBeOk = (expr: string) =>
+  Reducer.evaluate(expr)->Reducer_Helpers.rRemoveDefaultsExternal->E.R.isOk->expect->toBe(true)
+
+let registry = FunctionRegistry_Library.registry
+let examples = E.A.to_list(FunctionRegistry_Core.Registry.allExamples(registry))
+
+describe("Fn auto-testing", () => {
```

Comment:
todo: Make ``describe`` block for entire thing.

Reply:
As long as tests prove the point, nothing else matters :)